includes/admin_lists.php

```diff
@@ -1,41 +1,46 @@
 <?php
 
+// Rebuild tsml cache when using bulk actions (edit status or trash)
+add_action('transition_post_status', function ($new_status, $old_status, $post) {
+    if ($post->post_type === 'tsml_meeting') {
+        tsml_cache_rebuild();
+    }
+}, 10, 3);
+
 # Adding region dropdown to filter
-add_action('restrict_manage_posts', function () {
+add_action('restrict_manage_posts', function ($post_type) {
 
-    if (@$_GET['post_type'] != 'tsml_meeting') {
+    if( $post_type !== 'tsml_meeting' ) {
```

Comment:
would be great to get this PSR12 formatted with https://marketplace.visualstudio.com/items?itemName=bmewburn.vscode-intelephense-client - otherwise there will be unrelated whitespace changes on the next PR